### PR TITLE
Prevent Unfinished Submissions from Showing Up in Submissions Table

### PR DESCRIPTION
**Pull request by cameronblandford**
* Self-explanatory
* Also, makes a minor text-change to a column header, as requested by Hanh

### CODE REVIEW

File: app/controllers/submissions/index.js [hunk 1]
```diff
@@ -31,36 +35,88 @@ export default Controller.extend({
...
   }),
 
-  piColumns: [
-    { propertyName: 'publication', title: 'Article', component: 'submissions-article-cell' },
-    { title: 'Award Number (Funder)', component: 'submissions-award-cell' },
-    { propertyName: 'user', title: 'Corr. Author', component: 'submissions-author-cell' },
-    { propertyName: 'repositories', title: 'Repo', component: 'submissions-repo-cell' },
-    { propertyName: 'userSubmittedDate', title: 'Last Update Date', component: 'date-cell' },
-    { propertyName: 'userSubmittedDate', title: 'Submitted Date', component: 'date-cell' },
+  piColumns: [{
```
Comment (cameronblandford) [inline app/controllers/submissions/index.js]: I clearly need to reconfigure my beautify plugin, sorry about these unnecessary changes.

**Comment by Rytiggy** [comment]
LGRTM
twerp-bowtie-v2
